buildSrc/src/main/groovy/jqwik.common-configuration.gradle

```diff
@@ -1,3 +1,8 @@
+plugins {
+	// See https://github.com/vlsi/vlsi-release-plugins/blob/master/plugins/gradle-extensions-plugin/README.md
+	id 'com.github.vlsi.gradle-extensions'
+}
+
```

Comment:
Could the syntax from below be used here:
```
project.plugins.apply('com.github.vlsi.gradle-extensions')
```
But I might be confusing different Gradle concepts...

Reply:
Gradle's recommendation is to mention all the plugins in the first `plugins` block, and they recommend `id(..)` syntax (see https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block ).

At the same time, applying plugins via `apply` call is discouraged (see https://docs.gradle.org/current/userguide/plugins.html#sec:old_plugin_application )

I do not follow why you suggest using deprecated techniques, especially when keeping #281 in mind.

In any case, I do not care regarding the exact syntax you prefer, and it is hard for me to understand the reasons you prefer `project.plugins.apply`, so feel free to adjust the code the way you like.

If you absolutely want me to do the change, of course, I can do it, however, it looks like a waste of time from my point of view.

Reply:
I wonder if the trade-offs are different in shared configurations?

Anyway, at least it should be consistent. Let's ask the creator of the shared configs:
@tmohme Why did you use `project.plugins.apply` instead of `plugins { id '...' }` in https://github.com/jlink/jqwik/pull/282?

Reply:
I believe, that was by mistake. If you check https://github.com/jlink/jqwik/pull/282/files, you'll see that the PR added a lot `plugins {..}` calls, and `project.plugins.apply` was just in two places. They beg for moving into `plugins{...}`

Reply:
In that case, I'd vote for moving everything to `plugins{..}` in this PR.

Reply:
To be honest, I don't remember for sure why I used the `project.plugins.apply` syntax in the convention plugins (as opposed to the plugins-block syntax everywhere else).
Most likely I've seen it somewhere in the gradle docs, maybe there was a limitation some time ago, maybe it was a simple mistake  . . .
Anyway, I don't see any advantages in keeping the `project.plugins.apply` syntax and also would also recommend the `plugins {...}` syntax now.

Reply:
@tmohme Thanks for clarifying!